js/controller/AppController.js

```diff
@@ -24,6 +24,10 @@ app.controller('AppController', function ($scope, $location, $http, $route, $log
     $rootScope.sidebar = {
         show: false
     };
+    $rootScope.navibar = {
+        show: true
```

Comment:
@pixelipo I have tried to animate it like files but I didn't found how it works in files.

It was my fist try :(